embed.php

```diff
@@ -1,18 +1,18 @@
 <!doctype html>
-<html lang="<?php print $lang; ?>" class="h5p-iframe">
+<html lang="<?php print esc_attr($lang); ?>" class="h5p-iframe">
```

Comment:
@SimenBergo Aren't `esc_attr`, `esc_html`, `esc_url`, and `absint` WordPress functions that should not be used in H5P core?